src/appleseed/renderer/modeling/bsdf/energycompensation.cpp

```diff
@@ -445,8 +449,12 @@ void AlbedoTable3D::write_table_to_image(const bfs::path& filename) const
         }
     }
 
-    EXRImageFileWriter writer;
-    writer.write(filename.string().c_str(), image);
+    const std::string file = filename.string();
+
+    OIIOImageFileWriter writer{ file.c_str() };
```

Comment:
Here (and everywhere else relevant) too.

Reply:
Done